Tests/AppStoreConnectClientTests/AppStoreConnectClientTests.swift

```diff
@@ -97,7 +101,7 @@ final class AppStoreConnectClientTests: XCTestCase {
         let apps = try await client.fetchVersions(for: app)
         XCTAssertEqual(1, apps.count)
         XCTAssertEqual(apps.first?.appStoreState, "ACCEPTED")
-        XCTAssertEqual(apps.first?.version, "1.1.1")
+        XCTAssertEqual(apps.first?.version, "Foo.Bar.Baz")
```

Comment:
need to use corresponding mock object here

Reply:
Fixed.